packages/KIcon/icons/icn-notification-bell.svg

```diff
@@ -1 +1,6 @@
-<svg width="33" height="32" viewBox="0 0 33 33" xmlns="http://www.w3.org/2000/svg"><g><path stroke="#A3BBCC" stroke-width="1.5" fill="none" fill-rule="evenodd" d="M24.5 19c-1.378 0-2.5-1.121-2.5-2.5V12c0-3.641-2.24-6.951-5.603-8.325C16.022 2.108 14.637 1 13 1c-1.612 0-3.024 1.131-3.4 2.675C6.239 5.049 4 8.358 4 12v4.5C4 17.879 2.878 19 1.5 19c-.276 0-.5.224-.5.5s.224.5.5.5h23c.276 0 .5-.224.5-.5s-.224-.5-.5-.5M16 20h-6c-.276 0-.5.224-.5.5 0 1.93 1.57 3.5 3.5 3.5s3.5-1.57 3.5-3.5c0-.276-.224-.5-.5-.5"/></g></svg>
+<svg width="16" height="18" viewBox="0 0 16 18" xmlns="http://www.w3.org/2000/svg">
+  <g fill-rule="evenodd">
+    <path fill="#7D91B3" class="bell-clapper" d="M9.74999955 15h-4.5c-.207 0-.375.168-.375.375 0 1.4475 1.1775 2.625 2.625 2.625s2.62499995-1.1775 2.62499995-2.625c0-.207-.16799995-.375-.37499995-.375"/>
```

Comment:
adding classes to these paths so they can be animated at a later time in the application 😉